spec/dummy/app/models/application_record.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationRecord < ActiveRecord::Base
+  self.table_name_prefix = 'dummy_'
```

Comment:
@mvgijssel the dummy's models are prefixed to prevent name collisions with models defined in the specs